README.md

```diff
@@ -2,47 +2,117 @@
 
 ## Partner Study
 #### Answers to the following partner-study can be found [here](https://reactjs.org/docs/glossary.html)
-  * Describe the following, Single Page Application, Compilers, Bundlers, Elements, Components, JSX, Package Mangers, CDN, Props and State. 
-  * Write all answers in answers.md
+  * Describe the following, writing all answer in `answers.md`
+    - Single Page Application,
+    - Compilers, 
+    - Bundlers,
+    - Elements,
+    - Components,
+    - JSX,
+    - Package Mangers,
+    - CDN, 
+    - Props and State.
 
 ### Other Useful Resources
 * [CodeSandbox](https://codesandbox.io/s/z667kplonx) that was used in the lecture video: 
 * [API Documentation](https://reactjs.org/docs/react-api.html)
 * [Documentation](https://facebook.github.io/react/docs/jsx-in-depth.html) on JSX
 
 ### Battle Plan
- * Objective: At this point you have become familiar with the DOM and have done some research on the wonderful world of HTML and CSS. Now we're going to dive into modern front-end JavaScript development by learning about ReactJS.
- * You're going to be building a ToDo App (please hold your applause).
- * We know this seems trivial, but the best part about this assignment is that is shows off some of the strengths of React and you can take it as far as want.
+ * **Objective:** 
+   - At this point  you have become familiar with the  DOM and have done
+     some research on the wonderful world of HTML and CSS.
+   - Now  we're   going  to  dive  into   modern  front-end   JavaScript
+     development by learning about ReactJS.
```

Comment:
With GH flavored markdown, I'd rather not force the line breaks. This pertains to all your edits following.

I will leave it to @ryanhca and @seanchen1991 to evaluate the content changes